Makefile

```diff
@@ -17,9 +17,9 @@ clean:
 	rm -rf mu.egg-info
 	rm -rf .coverage
 	rm -rf docs/_build
-	find . \( -name '*.py[co]' -o -name dropin.cache \) -print0 | $(XARGS) rm
-	find . \( -name '*.bak' -o -name dropin.cache \) -print0 | $(XARGS) rm
-	find . \( -name '*.tgz' -o -name dropin.cache \) -print0 | $(XARGS) rm
+	find . \( -name '*.py[co]' -o -name dropin.cache \) -delete
+	find . \( -name '*.bak' -o -name dropin.cache \) -delete
+	find . \( -name '*.tgz' -o -name dropin.cache \) -delete
```

Comment:
The reason I had to change this is because on windows the `rm` application from MYSYS complained about trying to execute it without arguments.
Anyway, even though this would be slightly slower (since there would be no parallelism at all), it's not really an issue for this projects.